worlds/kh2/docs/en_Kingdom Hearts 2.md

```diff
@@ -63,6 +63,21 @@ For example, if you are fighting Roxas, receive Reflect Element, then die mid-fi
 - Customize the amount and level of progressive movement (Growth Abilities) you start with.
 - Customize start inventory, i.e., begin every run with certain items or spells of your choice.
 
+<h2 style="text-transform:none";>What are Lucky Emblems?:</h2>
+Lucky Emblems are items that are required to beat the game if your goal is "Lucky Emblem". <br>
+Think of these "McGuffins" as you need X amount of Proof of Nonexistences to open the final door .
```

Comment:
```suggestion
You can think of these as requiring X number of Proofs of Nonexistence to open the final door.
```
"McGuffin" is probably best avoided, since people won't know what it means, and the randomizer meaning is more nuanced/specific than the general meaning, "as you need" is grammatically incorrect, "amount" is used for non-countable nouns, whereas these are countable and should use "number", and the plural of "Proof of Nonexistence" would be "Proofs of Nonexistence"